pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -200,7 +202,8 @@ public String recommendConfig(String inputStr) {
   @Produces(MediaType.APPLICATION_JSON)
   @Path("/tables")
   @ApiOperation(value = "Lists all tables in cluster", notes = "Lists all tables in cluster")
-  public String listTableConfigs(@ApiParam(value = "realtime|offline") @QueryParam("type") String tableTypeStr) {
+  public String listTableData(@ApiParam(value = "realtime|offline") @QueryParam("type") String tableTypeStr,
+      @ApiParam(value = "asc|desc") @QueryParam("sortByTime") @DefaultValue("none") String sortOrderByTime) {
```

Comment:
Let's make it 2 parameters, one for the sort order (asc/desc, can also be modeled as a boolean), one for the sort type (name/time etc.)

Reply:
Sure